patches/go-1.22/1001-all-updated-vendored-x-net-with-security-fix.patch

```diff
@@ -0,0 +1,66 @@
+From 157d68b686ec5a5b4bfa8c871128f3e8406932f8 Mon Sep 17 00:00:00 2001
+From: Damien Neil <dneil@google.com>
+Date: Wed, 26 Feb 2025 16:08:57 -0800
+Subject: [PATCH] all: updated vendored x/net with security fix
+
+Fixes CVE-2025-22870
+
+(cherry picked from commit 25177ecde0922c50753c043579d17828b7ee88e7)
+
+[bcressey: backport to Go 1.22]
```

Comment:
Is this the last release of the SDK that will include Go 1.22 given that its EOL?